src/wp-content/themes/twentytwelve/js/navigation.js

```diff
@@ -25,31 +59,34 @@
 			menu.className = 'nav-menu';
```

Comment:
I know this is what the code originally did, but perhaps we want to add a class instead of replacing all the classes that may be in place already?

If so, we'd instead want to do:
```suggestion
			menu.classList.add( 'nav-menu' );
```